docs/src/vim.md

```diff
@@ -442,15 +442,15 @@ Here's an example of these settings changed:
 
-| Property                | Description                                                                                                                                                   | Default Value          |
-| ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------- |
-| cursor_blink            | If `true`, the cursor blinks.                                                                                                                                 | `true`                 |
-| relative_line_numbers   | If `true`, line numbers in the left gutter are relative to the cursor.                                                                                        | `true`                 |
-| scrollbar               | Object that controls the scrollbar display. Set to `{ "show": "never" }` to hide the scroll bar.                                                              | `{ "show": "always" }` |
-| scroll_beyond_last_line | If set to `"one_page"`, allows scrolling up to one page beyond the last line. Set to `"off"` to prevent this behavior.                                        | `"one_page"`           |
-| vertical_scroll_margin  | The number of lines to keep above or below the cursor when scrolling. Set to `0` to allow the cursor to go up to the edges of the screen vertically.          | `3`                    |
-| gutter.line_numbers     | Controls the display of line numbers in the gutter. Set the `"line_numbers"` property to `false` to hide line numbers.                                        | `true`                 |
-| command_aliases         | Object that defines aliases for commands in the command palette. You can use it to define shortcut names for commands you use often. Read below for examples. | `{}`                   |
+| Property                | Description                                                                                                                                                   | Default Value      |
+| ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------ |
+| cursor_blink            | If `true`, the cursor blinks.                                                                                                                                 | `true`             |
+| relative_line_numbers   | If `true`, line numbers in the left gutter are relative to the cursor.                                                                                        | `false`            |
+| scrollbar               | Object that controls the scrollbar display. Set to `{ "show": "never" }` to hide the scroll bar.                                                              | `{ "show": null }` |
```

Comment:
`scrollbar.show` defaults to `auto` I believe.
https://github.com/zed-industries/zed/blob/39fdb1efb7a227ca926c4d7dfae0e824d9e29a95/assets/settings/default.json#L245

The real default is actually actually:
```
{"show": "auto", "cursors": true, "git_diff": true, "search_results": true, "selected_symbol": true, "diagnostics": true}
```
But I think that would format poorly so I think `{ "show": "auto" }` would be fine.

Reply:
Sorry, it's my mistake. I blindly take the default value of `project_panel.scrollbar`. I should have double-checked that.